projects/plugins/crm/src/js/components/automations-admin/types.tsx

```diff
@@ -0,0 +1,37 @@
+export type Trigger = {
+	slug: string;
+	title: string;
+	description?: string;
+	category: Category;
+};
+
+export type Step = {
+	attributes: string[];
+	nextStep?: Step;
+	slug: string;
+	title: string;
+	description: string;
+	type: Type;
+	category: Category;
+	allowedTriggers: Trigger[];
+};
+
+export type Action = Step;
+
+export type Condition = Step;
+
+export type Workflow = {
+	id: number;
+	name: string;
+	description: string;
+	category: Category;
+	triggers: Trigger[];
+	initial_step: Step;
+	active: boolean;
+	version: number;
+	added: string;
```

Comment:
Did we add a task to add `added` in our PHP code?

Reply:
I briefly looked for such a task and did not see one, but I believe you were going to create one if it didn't exist.